script/add-payment-partials-to-marriage-abroad-services-data.rb

```diff
@@ -1,124 +1,14 @@
 countries = %w(
-  afghanistan
-  albania
-  algeria
-  american-samoa
-  andorra
-  angola
-  armenia
-  aruba
-  austria
-  azerbaijan
-  bahrain
-  benin
-  bhutan
-  bolivia
-  bonaire-st-eustatius-saba
-  bosnia-and-herzegovina
-  brazil
   bulgaria
-  burkina-faso
-  burundi
-  cape-verde
-  central-african-republic
-  chad
-  chile
-  comoros
-  congo
   croatia
-  cuba
-  curacao
-  democratic-republic-of-congo
-  denmark
-  djibouti
-  el-salvador
-  equatorial-guinea
-  eritrea
-  estonia
-  ethiopia
-  finland
-  gabon
-  georgia
-  greece
+  cyprus
   guatemala
-  guinea
-  guinea-bissau
-  haiti
-  honduras
-  hungary
-  iceland
-  iraq
-  jordan
-  kazakhstan
-  kyrgyzstan
-  latvia
-  libya
-  liechtenstein
-  lithuania
-  luxembourg
-  macedonia
-  mali
-  marshall-islands
-  mauritania
-  mexico
-  micronesia
   moldova
-  montenegro
-  nepal
-  nicaragua
-  niger
-  norway
-  palau
   panama
-  paraguay
-  romania
-  russia
-  rwanda
-  san-marino
-  sao-tome-and-principe
-  serbia
-  south-sudan
-  st-maarten
-  sudan
-  suriname
-  sweden
-  tajikistan
-  timor-leste
-  togo
-  tunisia
-  turkmenistan
-  uzbekistan
   venezuela
-  western-sahara
 )
 
-# Exclude #country_without_consular_facilities? countries
-countries = countries -
-  SmartAnswer::Calculators::MarriageAbroadDataQuery::COUNTRIES_WITHOUT_CONSULAR_FACILITIES
-
-# Excluded countries ignored in outcome template
-countries = countries -
-  %w(cote-d-ivoire burundi)
-
 payment_method_partial = {
-  'armenia' => 'pay_in_local_currency_ceremony_country_name',
-  'bosnia-and-herzegovina' => 'pay_in_local_currency_ceremony_country_name',
-  'cambodia' => 'pay_in_local_currency_ceremony_country_name',
-  'iceland' => 'pay_in_local_currency_ceremony_country_name',
-  'latvia' => 'pay_in_local_currency_ceremony_country_name',
-  'slovenia' => 'pay_in_local_currency_ceremony_country_name',
-  'tunisia' => 'pay_in_local_currency_ceremony_country_name',
-  'tajikistan' => 'pay_in_local_currency_ceremony_country_name',
-
-  'kazakhstan' => 'pay_by_cash_in_kazakhstan',
-  'kyrgyzstan' => 'pay_by_cash_in_kazakhstan',
-
-  'luxembourg' => 'pay_in_cash_visa_or_mastercard',
-
-  'russia' => 'pay_by_mastercard_and_visa_only',
-
-  'finland' => 'pay_in_euros_or_visa_electron',
-
```

Comment:
Hopefully the PR description and commit notes of PR #2507 make the purpose of this script clearer!